config/katello.migrations/201117205851-add-import-export-params.rb

```diff
@@ -0,0 +1,8 @@
+if answers['pulpcore'].is_a?(Hash)
+  answers['pulpcore']['allowed_import_path'] = ['/var/lib/pulp/sync_imports', '/var/lib/pulp/imports']
+  answers['pulpcore']['allowed_export_path'] = ['/var/lib/pulp/exports']
```

Comment:
Just a general note to help with learning some of this, the `pulpcore` module is not exposed in the answers file and therefor changing them won't achieve anything. The pulpcore module is wrapped by the `puppet-foreman_proxy_content` module as I think you've encountered looking at your other PRs and thus all configuration has to flow through it or through hiera (an advanced method).

Reply:
@ehelms So I guess that means I don't need the first section of this migration, and I'll leave only the second.